src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedParameterSymbol.cs

```diff
@@ -136,6 +139,11 @@ public override ImmutableArray<SyntaxReference> DeclaringSyntaxReferences
             }
         }
 
+        internal bool IsNullChecked
```

Comment:
I know the existing code uses fields and explicit properties. but that's just because it was written pre-C# 6. You can use a get-only autoprop here, i.e.

`internal bool IsNullChecked { get; }`